npk/src/npk.rs

```diff
@@ -848,19 +863,19 @@ fn write_npk<W: Write + Seek>(
             context: "Could create aligned zip-file".to_string(),
             error: e,
         })?;
-    let mut fsimg = File::open(&fsimg).map_err(|e| Error::Os {
+    let fsimg = File::open(&fsimg).await.map_err(|e| Error::Os {
         context: format!("Failed to open '{}'", &fsimg.display()),
         error: e,
     })?;
 
-    io::copy(&mut fsimg, &mut zip).map_err(|e| Error::Os {
+    io::copy(&mut fsimg.into_std().await, &mut zip).map_err(|e| Error::Os {
```

Comment:
probably this should be `tokio::io::copy` instead

Reply:
The second argument is unfortunately a synchronous zip file.